airflow/providers/google/cloud/transfers/salesforce_to_gcs.py

```
@@ -121,5 +126,22 @@ def execute(self, context: Dict):
             )
 
             gcs_uri = f"gs://{self.bucket_name}/{self.object_name}"
-            self.log.info("%s uploaded to GCS", gcs_uri)
-            return gcs_uri
+            return_val = {'data': gcs_uri}
+            self.log.info("Data file: %s uploaded to GCS", gcs_uri)
+
+            # Writing schema only works if a salesforce_object is defined
```

> Writing schema only works if a salesforce_object is defined

then i'd raise for this combination in init so doesnt just silently fail